OCAT/index.js

```diff
@@ -25,6 +25,7 @@ app.use(`/public`, (req, res) => {
 
 RouteLoader(app)
   .then(() => {
+    app.post(`/assessment`, AssessmentSubmitRoute);
```

Comment:
```suggestion
```
you shouldn't need this.  The RouteLoader function should do this automatically.  let me know if it isn't

Reply:
I removed it and it works.